README.md

```diff
@@ -107,32 +91,25 @@ channels:
 
 ## Working Groups And Teams
 
+The CNCF tag-contributor-strategy has working groups and teams to effectively collaborate and coordinate efforts towards achieving common goals and objectives, as well as to distribute tasks and responsibilities among its members for efficient execution. This approach allows for a more streamlined and organized approach to managing complex projects within the CNCF ecosystem.
+
 ### Governance WG
 
 [Charter](/governance/README.md)
 
-Facilitators:
-
-- Josh Berkus ([@jberkus](https://github.com/jberkus)), Red Hat
-- Dawn Foster ([@geekygirldawn](https://github.com/geekygirldawn)), VMware
-
 ### Contributor Growth WG
 
 [Charter](/contributor-growth/README.md)
 
-Facilitators:
+### Mentoring WG
 
-- Carolyn Van Slyck ([@carolynvs](https://github.com/carolynvs)), Microsoft
-- Riaan Kleinhans ([@Riaankl](https://github.com/Riaankl)), ii
+[Charter](/mentoring/README.md)
 
 ### Maintainer's Circle
 
-Status: [Proposed](https://github.com/cncf/tag-contributor-strategy/issues/1)
+[Charter](/maintainers-circle/README.md)
 
-Facilitators:
-
-- Dave Sudia ([@thedevelopnik](https://github.com/thedevelopnik)), Ambassador Labs
-- Paris Pittman ([@parispittman](https://github.com/parispittman)), Apple
+Status: [Proposed](https://github.com/cncf/tag-contributor-strategy/issues/1)
 
 ### Inclusiveness Team
```

Comment:
This isn't consistent with the other teams (listing out the facilitators directly on the readme) and also the group still in the proposed stage.

I have created an issue to capture what we were displaying on the readme, and would like to see us remove inclusiveness from the readme since it's not accepted or active yet. https://github.com/cncf/tag-contributor-strategy/issues/365

Reply:
Updated. Thanks